src/api/libcellml/generatorprofile.h

```diff
@@ -2655,9 +2697,7 @@ class LIBCELLML_EXPORT GeneratorProfile
      * @brief Set the @c std::string for the interface to create the states
      * array.
      *
-     * Set this @c std::string for the interface to create the states array. To
-     * be useful, the string should contain the <CODE> tag, which will be
-     * replaced with some code to create the states array.
+     * Set this @c std::string for the interface to create the states array.
      *
```

Comment:
I don't understand the "this"? Should it be "the"?

Reply:
Yes, typo. Fixed in commit f1d0e48d.

Reply:
See @kerimoyle, this is what I was talking about: I made local changes, got a commit id from GitHub Desktop (f1d0e48d in this instance), made [a comment](https://github.com/cellml/libcellml/pull/631#discussion_r468945178), resolved the conversation. Repeated this for other conversations. Then, pushed all my changes. Now, I would have expected f1d0e48d to be a link in [my comment](https://github.com/cellml/libcellml/pull/631#discussion_r468945178), but it's not...?!

Reply:
It's because you've included a trailing "d" that shouldn't be there?

Reply:
I thought of that last time, but that's the commit id I get from GitHub Desktop and it certainly works... checking commit f1d0e48d (check the source to confirm that I did indeed use `f1d0e48d`).